challenge-04/README.md

```diff
@@ -7,15 +7,55 @@ um único parâmetro como argumento. Essa função deve retornar `true` se o
 equivalente booleano para o valor passado no argumento for `true`, ou `false`
 para o contrário.
 */
-?
+
+var isTruthy = function(x){
+	if (x){
+		return true;
+	}
+	else{
```

Comment:
-1
Como você faz o `return` no `if`, o `else` nem é necessário, pois se o `if` for falso, ele já vai pular e executar tudo o que vier abaixo ;)